website/docs/plugin/framework/functions/testing.mdx

```diff
@@ -215,7 +215,9 @@ func TestEchoFunctionRun(t *testing.T) {
         t.Run(name, func(t *testing.T) {
             t.Parallel()
 
-            got := function.RunResponse{}
+            got := function.RunResponse{
+                Result: function.NewResultData(basetypes.StringValue{}),
+            }
```

Comment:
nit: we should be able to use an unknown here, since that mimics our internal logic
```suggestion
            got := function.RunResponse{
                Result: function.NewResultData(types.StringUnknown()),
            }
```